docs/coding/javascript.md

```diff
@@ -78,89 +79,72 @@ do {
 ```
 These should all be followed by a semi-colon. In addition the `return` value expression must start on the same line as the return keyword in order to avoid semi-colon insertion.
 
-If the "Optimize JavaScript files" performance option in Drupal 6 is enabled, and there are missing semi-colons, then the JS aggregation will fail. It is therefore very important that semi-colons are used.
+- :heavy_check_mark: DO use a semi-colon except where it's not allowed
+- :heavy_check_mark: DO configure ESLint rule `semi` with option `["error", "always"]`.
 
 ## Control Structures
 These include `if`, `for`, `while`, `switch`, etc. Here is an example if statement, since it is the most complicated of them:
 ```js
-if (condition1 || condition2) 
-{
+if (condition1 || condition2) {
   action1();
 }
-else if (condition3 && condition4) 
-{
+else if (condition3 && condition4) {
   action2();
 }
-else 
-{
+else {
   defaultAction();
 }
 ```
+
 Control statements should have a new line between the control keyword and opening parenthesis, to distinguish them from function calls.
```

Comment:
Thank you for making this change. We'll need to update the description to match, though.

```suggestion
Control statements should have single spaces between the control keyword, control condition, and opening parenthesis.
```